java/app-encryption/src/test/java/com/godaddy/asherah/appencryption/persistence/DynamoDbMetastoreImplTest.java

```diff
@@ -216,17 +218,54 @@ void testStoreWithFailureShouldThrowException() {
   void testPrimaryBuilderPath() {
     // Hack to inject default region since we don't explicitly require one be specified as we do in KMS impl
     System.setProperty(SDKGlobalConfiguration.AWS_REGION_SYSTEM_PROPERTY, "us-west-2");
+    DynamoDbMetastoreImpl dynamoDbMetastoreImpl = DynamoDbMetastoreImpl.newBuilder().build();
+
+    assertNotNull(dynamoDbMetastoreImpl);
+  }
+
+  @Test
+  void testBuilderPathWithRegion() {
+    DynamoDbMetastoreImpl dynamoDbMetastoreImpl = DynamoDbMetastoreImpl.newBuilder().withRegion("us-west-2").build();
 
-    DynamoDbMetastoreImpl dynamoDbMetastoreImpl =
-      DynamoDbMetastoreImpl.newBuilder().build();
     assertNotNull(dynamoDbMetastoreImpl);
   }
 
   @Test
   void testMetastoreWithRegionSuffix() {
-    DynamoDbMetastoreImpl dynamoDbMetastore = DynamoDbMetastoreImpl.newBuilder().withDynamoDbRegionSuffix("us-west-2").build();
+    DynamoDbMetastoreImpl dynamoDbMetastore = DynamoDbMetastoreImpl.newBuilder()
+      .withDynamoDbRegionSuffix("us-west-2")
+      .build();
 
     assertEquals("", dynamoDbMetastoreImpl.getRegionSuffix());
     assertEquals("us-west-2", dynamoDbMetastore.getRegionSuffix());
   }
+
+  @Test
+  void testWithTableName() {
```

Comment:
The name of this test can be changed to something meaningful. A lot is happening in this test.

I believe this test creates a client using standard builder provided by AWS and then you create a table in it.
Later you verify that the library is able to connect to the same table, which got created in the previous step.